dist/.ebextensions/01_bridgecf.config

```diff
@@ -60,16 +61,16 @@ files:
         redis.min.idle = ${REDIS_MIN_IDLE:-3}
         redis.max.idle = ${REDIS_MAX_IDLE:-50}
         redis.timeout = ${REDIS_TIMEOUT:-2000}
-        redis.url = ${REDIS_URL:-redis://provider:password@localhost:6379}
+        redis.url = ${REDIS_URL:-redis://AWS:AWS@localhost:6379}
```

Comment:
Initially, we want both AWS and Heroku to point to the same Redis instance. Trying to migrate both at once will break our session handling.